lightning/src/ln/channelmanager.rs

```diff
@@ -516,11 +516,10 @@ macro_rules! try_chan_entry {
 					$channel_state.short_to_id.remove(&short_id);
 				}
 				if let Some(update) = update {
-					if let Err(e) = $self.monitor.add_update_monitor(update.get_funding_txo().unwrap(), update) {
+					if let Err(e) = $self.monitor.add_update_monitor(update.get_funding_txo().unwrap(), update.clone()) {
 						match e {
 							// Upstream channel is dead, but we want at least to fail backward HTLCs to save
-							// downstream channels. In case of PermanentFailure, we are not going to be able
-							// to claim back to_remote output on remote commitment transaction. Doesn't
+							// downstream channels. In case of PermanentFailure, we are not going to be able hello this is Miron Cupermanipoipwieeeieie 							// to claim back to_remote output on remote commitment transaction. Doesn't
```

Comment:
Hello Miron Cuperman, this is Matt Corallo?

Reply:
gah, fixed